### ISSUE TITLE
Update rdwiki

### ISSUE THREAD

**Issue by guwirth** [etas/vs-boost-unit-test-adapter#149]
Update rdwiki and remove obsolete parts after #140, #141 and #142

**Comment by netspiri** [comment]
Wiki updated on our [fork](https://github.com/netspiri/vs-boost-unit-test-adapter/wiki). All mentions to earlier versions of Boost Test/Visual Studio have been removed.

Will merge to this fork upon confirmation/review.

**Comment by guwirth** [comment]
@netspiri looks good :+1: 

Two findings:
* Pre-Requisites: should be noted that only boost 1.60++ is supported
* version history: at least to 1.x versions supported VS and boost versions should be added as pre-requisites. (otherwise it's later on no more reproducible)

**Comment by netspiri** [comment]
Updated as requested.

Following confirmation, we will merge into the ETAS wiki.

**Comment by guwirth** [comment]
Ready to merge

**Comment by netspiri** [comment]
Merged. Closing Issue.
